amortized_lda.ipynb

```diff
@@ -0,0 +1,925 @@
+{
```

Comment:
<p>I would try with the topic latent space here, using correlation or hellinger distance metric</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
sure, should we just show both of them? I have seen this way of presenting topics more often than a UMAP on the topic latent space in general

Reply:
both is good, you can use `sc.pl.embedding` to display arbitrary obsm, need to copy the umap with a different name